test/fixtures/hello-cloud/stateful_set.yml

```diff
@@ -11,9 +11,9 @@ spec:
         app: hello-cloud
         name: nginx-ss
     spec:
+      terminationGracePeriodSeconds: 0
       containers:
       - name: nginx
-        image: gcr.io/google_containers/nginx-slim:0.8
-        command: ["sleep", "40"]
```

Comment:
This is actually the biggest speed improvement, getting the `apply` speed down around 12s. I believe sleep doesn't handle SIGTERM, so we end up waiting for the pod to die or hit the default termination grace period.